media/js/firefox/new/experiment-download-scene2-email.js

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment-download-scene2-email-082018',
+        variations: {
+            'v=a': 50,
```

Comment:
Which is the control group here? The new page isn't made default, but we're still putting 100% of users into the copy A/B test. Usually here we'd add a control group to compare conversions against, which normally defaults to the current experience in prod. It feels like that's not what we're testing here though, we're just testing the copy. Is there any reason why we can't make this new page default regardless of the experiment? Then we could treat say, version a, as the control?